src/components/Form/Checkbox.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { ErrorText } from './ErrorText';
+import { Input } from './Input';
+import { Label } from './Label';
+
+type Props = {
+  error?: string;
+  label?: string;
+  name: string;
+  disabled?: boolean;
+};
+
+const Checkbox: React.FunctionComponent<Props> = ({ error, name, label, ...props }: Props) => {
+  return (
+    <CheckContainer disabled={props.disabled}>
+      <Input id={name} name={name} {...props} />
+      <Label htmlFor={name}>{label}</Label>
+      {error && <ErrorText>{error}</ErrorText>}
+    </CheckContainer>
+  );
+};
+
+const CheckContainer = styled.div<{ disabled?: boolean }>`
```

Comment:
There is an issue with the box-sizing, compared to the original one + the transition doesn't do a whole lot